_pages/business-units/18f/how-18f-works/18f-project-lifecycle.md

```diff
@@ -0,0 +1,88 @@
+---
+title: High-level 18F project lifecycle
+questions:
+  - 18f-bd
+  - 18f-staffing
+---
+
+Projects at 18F begin as opportunities. An opportunity can arrive in a variety
+of ways. For example, a potential partner agency can contact us directly to ask
+about working together, the director of business development or a portfolio
+director may reach out to an agency to follow-up on a recent news story that
+indicates a need we may be able to help with, or a previous partner agency that
+was happy with our work together spreads the word to their contacts in other
+agencies.
+
+However it happens, when a new opportunity comes in, the
+[business development]({{site.base_url}}/18f-bd/) process kicks off. To help
+keep track of it, it is added to the
+[BD Opportunities tab in the 18F Dashboard🔒](https://airtable.com/tblXEgAIZzipuVNZY/viwoNkShDjSTt2Adu?blocks=hide).
+The new opportunity will begin with a stage of **Identified**.
+
+Depending on the category of the opportunity, the director of business
+development, a portfolio director, or a domain champion will then take over as
+the BD lead to shepherd it through the process. They will look into the lead,
+doing some background research to understand the need and talk to project leads,
+account managers, and other folks who have any history with the partner or have
+knowledge or experience with the project or space. Once they feel like they have
+a good initial idea of the need, the Dashboard is updated and the opportunity
+is moved into the **Prioritized** stage.
+
+The BD lead will schedule a meeting with the potential partner agency, the
+business development reps from the chapters, an account manager, and appropriate
+portfolio directors. This meeting is about making sure 18F understands the
+agency's needs, that our offerings fit their needs, that we have the right
+expertise to help them, and that the project fits with what we value. After this
+meeting, if everyone on the 18F team agrees that we should move forward, the
+Dashboard is updated again to move the opportunity into the **Qualified** stage,
+meaning that we believe it's a good project that we should try to take on.
+
+Once an opportunity is qualified, the account manager writes up a scope of work,
```

Comment:
Account managers usually write a short (1-2 pages) plain language proposal first, to align on the need, the desired outcomes, and the general shape of the work. The SOW comes later, and includes more detail about terms of the partnership.

Reply:
Ooh, thank you! The SOW is part of the IAA package, I guess?

Reply:
Yep, that's right.